Directory.Packages.props

```diff
@@ -38,9 +38,9 @@
     <PackageVersion Include="Pgvector" Version="0.2.0" />
     <PackageVersion Include="Pgvector.EntityFrameworkCore" Version="0.2.0" />
     <!-- Version together with runtime -->
-    <PackageVersion Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="$(RuntimeVersion)" />
-    <PackageVersion Include="Microsoft.Extensions.Options" Version="$(RuntimeVersion)" />
```

Comment:
Extensions packages from dotnet/runtime repo don't rev versions at the same time as each other. And since we set `CentralPackageTransitivePinningEnabled` we're bringing in different versions of some dependencies.

In this specific case, `Microsoft.Extensions.Http.Resilience` 8.1.0 brings in 8.0.1 of `Microsoft.Extensions.Options` which conflicts with 8.0.0. 8.0.0 is still the latest version of `Microsoft.Extensions.Options.ConfigurationExtensions` and `Microsoft.Extensions.DependencyInjection.Abstractions` so we can't use the same version for all extensions packages from runtime.